.github/scripts/compare.ts

```
@@ -0,0 +1,508 @@
+  baseBundle: BundleData,
+  minimumChangeThreshold?: number
+): { newPages: PageChange[]; changedPages: PageChange[] } => {
+  const pages = Object.entries(currentBundle)
+    .filter(([page]) => page !== "__global")
+    .reduce<{ newPages: PageChange[]; changedPages: PageChange[] }>(
+      (acc, [page, currentStats]) => {
+        const baseStats = baseBundle[page];
+
+        if (!baseStats) {
+          return {
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)